server/bin/www

```diff
@@ -43,6 +43,7 @@ io.on("connection", (socket) => {
   socket.on("new-message", (data) => {
     socket.broadcast.emit("new-message", {
       message: data.message,
+      recipientId: data.recipientId,
       sender: data.sender,
     });
```

Comment:
This still sends the socket message to every connected socket, so it wouldn't deal with the socket issue. But you don't have to worry about the sockets until issue 4.